.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Handle Release
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+        
+      - name: Set GOPATH
+      # temporary fix
+      # see https://github.com/actions/setup-go/issues/14
+        run: |
+          echo "##[set-env name=GOPATH;]$(dirname $GITHUB_WORKSPACE)"
+          echo "##[add-path]$(dirname $GITHUB_WORKSPACE)/bin"
+        shell: bash
+    
+      - name: Prepare environment
+        run: |
+          go mod download
+          hack/install-gox.sh\
+
+      - name: Install dependencies
+        run: |
+          go mod tidy && git diff --no-patch --exit-code
+          hack/verify-code-patterns.sh
+          hack/verify-boilerplate.sh
+          hack/run-lint.sh
+          go test -short -coverprofile=coverage.txt -covermode=atomic ./...
+          hack/make-all.sh
+          hack/verify-installation.sh
+          hack/ensure-kubectl-installed.sh
+          hack/run-integration-tests.sh
+          hack/verify-receipts-upgrade-migration.sh
+
```

Comment:
let's split these steps with proper names. :)

Reply:
It's done, you can check it.